ensime_shared/ensime.py

```diff
@@ -674,7 +676,9 @@ def send_refactor_request(self, ref_type, ref_params, ref_options):
 
     def apply_refactor(self, call_id, payload):
         """Apply a refactor depending on its type."""
-        if payload["refactorType"]["typehint"] in ["Rename", "InlineLocal", "AddImport", "OrganizeImports"]:
+        supported_refactorings = ["Rename", "InlineLocal", "AddImport", "OrganizeImports"]
```

Comment:
Good guess-best-intermediate-variable-name logic for a formatting tool :stuck_out_tongue_winking_eye: